### DISCUSSION TITLE
Rainmeter plugin question/suggestion

### DISCUSSION THREAD

**Discussion by janeknits** [seerge/g-helper#2495] [Ideas]
This is a Rainmeter issue rather than a G-Helper one so I hope it isn't considered off-topic.   Could a Rainmeter plugin be made to pull info from G-Helper?  I have no idea if that is a huge ask or not.   
I usually check G-Helper at start-up and then close the window until I need to change a setting.  Being able to list some settings info in my existing Rainmeter skins would be helpful.  The temperatures would be my first interest.  Right now Rainmeter needs either HWinFO or MSI Afterburner running to get the temperatures.   Are any Rainmeter folks here?

**Comment by seerge** [comment]
@janeknits hello. I'm afraid this is too specific and narrow case :) 

Temp readings that you use in Rainmeter aver very generic not unique to G-Helper, so why not keep using hwinfo (or may be find even lighter metrics app) that provide that ?
